src/components/pages/TagPage/TagPage.tsx

```diff
@@ -1,14 +1,11 @@
 import React from 'react';
-import { Typography } from '@material-ui/core';
-
 import { TagList } from '../../../containers/Tag/TagList/TagList';
 
 export interface TagPageProps {}
 
 export const TagPage: React.SFC<TagPageProps> = () => {
   return (
     <div>
-      <Typography variant="h5">Tags</Typography>
```

Comment:
Did this to make title on same line as search and `Add New` (just for design purposes). I can change back if needed.